frontend/src/components/Pages/MissionHistoryPage/HistoricMissionCard.tsx

```diff
@@ -42,6 +42,9 @@ export function HistoricMissionCard({ index, mission }: IndexedMissionProps) {
                     {mission.name}
                 </Button>
             </Table.Cell>
+            <Table.Cell>
+                <Typography>{mission.area?.areaName}</Typography>
```

Comment:
Do we want it to be empty when area is missing or do we want a placeholder value? If placeholder value is desired it can be achieved as shown below.

```suggestion
                <Typography>{mission.area ? mission.area.areaName : '-'}</Typography>
```